Mathlib/MeasureTheory/Group/Measure.lean

```diff
@@ -761,6 +784,21 @@ theorem isHaarMeasure_map [BorelSpace G] [TopologicalGroup G] {H : Type _} [Grou
 #align measure_theory.measure.is_haar_measure_map MeasureTheory.Measure.isHaarMeasure_map
 #align measure_theory.measure.is_add_haar_measure_map MeasureTheory.Measure.isAddHaarMeasure_map
 
+/-- The image of a Haar measure under right multiplication is again
+a Haar measure. -/
+theorem isHaarMeasure_rmul [BorelSpace G] [TopologicalGroup G] [T2Space G] (g : G)
+  : IsHaarMeasure (Measure.map (· * g) μ) :=
+{
+  toIsMulLeftInvariant := isMulLeftInvariant_rmul g
+  lt_top_of_isCompact := by
+    intro K hK
+    rw [map_apply (measurable_mul_const _) hK.measurableSet]
+    apply IsCompact.measure_lt_top _
+    rwa [← Homeomorph.coe_mulRight g, Homeomorph.isCompact_preimage _]
+  toIsOpenPosMeasure := by
+    exact (continuous_mul_right g).isOpenPosMeasure_map (mul_right_surjective g)
```

Comment:
Do the `by` and `exact` cancel out? One means "go from term mode to tactic mode" and the other means "go from tactic mode to term mode".

Reply:
Yep :)